custom/conf/app.example.ini

```diff
@@ -1687,6 +1687,16 @@ LEVEL = Info
 ;; convert \r\n to \n for Sendmail
 ;SENDMAIL_CONVERT_CRLF = true
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[mailer.override_header]
```

Comment:
I think it's better to call it `custom_header`, because:

1. Some headers are not override-able, eg: `Message-ID` in code below.
2. We never guarantee that everything here could succeed overriding: it depnds on a lot of facts, eg: the MTA behaviors.

So "custom header" seems more general, and could avoid some misleading.

Reply:
well I still think it should overwrite headers ... to satisfy you I make Message-ID overrride too

Reply:
But why `Message-ID` should be override-able? It should be unique for a thread.

Reply:
yes, but we already agreed that this setting option should en-power the sysadmin to do whatever email stuff he has to do ... If he wants to create broken mails, he can do so ... why should you care?

-> https://github.com/go-gitea/gitea/pull/27860#issuecomment-2136270496 ...